code/modules/power/tesla/energy_ball.dm

```diff
@@ -346,13 +349,10 @@
 			tesla_zap(closest_mob, 5, power / 1.5, tesla_flags, shocked_targets, zap_gib)
 
 	else if(closest_machine)
-		closest_machine.tesla_act(power, tesla_flags, shocked_targets)
-		tesla_zap(closest_machine, 5, power / 1.1, tesla_flags, shocked_targets, zap_gib)
```

Comment:
removing this will make the tesla_zap uses the same power reduction as in tesla_act()